diff --git a/node_modules/react-native-iap/android/build.gradle b/node_modules/react-native-iap/android/build.gradle
index e4a79cc..a7e1d16 100644
--- a/node_modules/react-native-iap/android/build.gradle
+++ b/node_modules/react-native-iap/android/build.gradle
@@ -58,18 +58,6 @@ android {
     targetCompatibility JavaVersion.VERSION_1_8
   }
 
-  flavorDimensions "store"
-
-  productFlavors {
-    amazon {
-      dimension "store"
-    }
-
-    play {
-      dimension "store"
-    }
-  }
-
   testOptions {
     unitTests.all {
       jvmArgs '-noverify'
@@ -164,10 +152,8 @@ dependencies {
   testImplementation "junit:junit:4.13.2"
   testImplementation "io.mockk:mockk:1.13.5"
 
-  playImplementation "com.android.billingclient:billing-ktx:$playBillingSdkVersion"
-  playImplementation "com.google.android.gms:play-services-base:$playServicesVersion"
-
-  amazonImplementation  "com.amazon.device:amazon-appstore-sdk:$amazonSdkVersion"
+  implementation "com.android.billingclient:billing-ktx:$playBillingSdkVersion"
+  implementation "com.google.android.gms:play-services-base:$playServicesVersion"
 
   if (supportLibVersion && androidXVersion == null) {
     implementation "com.android.support:support-annotations:$supportLibVersion"
